frontend/webapp/configs/config.py

```diff
@@ -75,13 +77,24 @@ def configure_app(app):
     app.artemis_logger = logging.getLogger("webapp_logger")
 
     # Configure Security
-    user_datastore = SQLAlchemyUserDatastore(db, User, Role)
-    app.security = Security(
-        app,
-        user_datastore,
-        register_form=ExtendedRegisterForm,
-        login_form=ExtendedLoginForm,
-    )
+    if app.config["AUTH_METHOD"] == "ldap":
+        app.artemis_logger.info("LDAP login enabled")
+        user_datastore = LDAPUserDatastore(db, User, Role)
+        app.security = Security(
+            app,
+            user_datastore,
+            register_form=ExtendedRegisterForm,
+            login_form=LDAPLoginForm,
```

Comment:
does this login form differ from what we used in the alternative case?

Reply:
(I mean aesthetically)

Reply:
no